sklearn/covariance/tests/test_graph_lasso.py

```diff
@@ -131,3 +134,26 @@ def test_graph_lasso_cv(random_state=1):
 
     # Smoke test with specified alphas
     GraphLassoCV(alphas=[0.8, 0.5], tol=1e-1, n_jobs=1).fit(X)
+
+
+def test_deprecated_grid_scores(random_state=1):
+    dim = 5
+    n_samples = 6
+    random_state = check_random_state(random_state)
+    prec = make_sparse_spd_matrix(dim, alpha=.96,
+                                  random_state=random_state)
+    cov = linalg.inv(prec)
+    X = random_state.multivariate_normal(np.zeros(dim), cov, size=n_samples)
+    graph_lasso = GraphLassoCV(alphas=[0.8, 0.5], tol=1e-1, n_jobs=1)
+    graph_lasso.fit(X)
+
+    depr_message = ("Function grid_scores is deprecated; Attribute grid_scores"
```

Comment:
:\ indeed. I disagree with those docs. "Function x is deprecated" is very confusing to users for a non-function attribute, isn't it?